### ISSUE TITLE
Request: Suggested migration from seanmcbreen.spell

### ISSUE THREAD

**Issue by seanmcbreen** [Jason-Rev/vscode-spell-checker#69]
Hi,

I'm the author of: https://marketplace.visualstudio.com/items?itemName=seanmcbreen.Spell and a member of the VS Code team.

The backing service for the extension has been failing for some time now and I've reached the point where I really have to remove it.  As part of this I was thinking of pointing user to your extension.  

I'm imagining sending out an update that suggests via an info message to uninstall and use your one - and then later on an update removing all functionality - eventually resulting in unpublishing the extension.

Are you ok with that?  I imagine it could give you an influx of users (as well as some new issues).

To be completely transparent we are looking to include additional support for spell checking in the core VS Code product at some point but for now I think your extension is the best one out there.

Sean

**Comment by Jason3S** [comment]
Hi Sean,

I feel honored.  I think your extension has been an inspiration for many people.

This extension is a fun side project.  I moved to VS Code from IntelliJ.  VS Code is 1000x better with Typescript.  The thing I missed the most was a spell checker.  I tried your extension first, but then realized it was for plain text and markdown.  So, I used yours as an example on how to write my own.  Thank you for that.

I would welcome the influx of users.  I think the basis is now in place.  Most of the improvement have to do with ease of configuration and adding new languages.

BTW: I used to work for Microsoft.  I was there from 1995 through 2003 when I left to travel the world.

**Comment by seanmcbreen** [comment]
Ok, that's great news - I may look at starting this process tomorrow.  

That said I'm actually on Holiday next week so may run out of time.

I really like many of the cool innovations in your extension so I think people will enjoy using it.

I'll close this issue - and update it once I publish the next update.

Thank you for supporting VS Code!
Sean

**Comment by seanmcbreen** [comment]
Sorry for the slow reply here - we are currently working on this for the core product.  As a result I've held off any change to my extension.  

If we do add to the core I think that is my preferred migration path for users.

A good discussion of this is happening over here:
https://github.com/Microsoft/vscode/issues/20266

This happened faster that I was thinking it would.  So I'm sorry if I messed you around at all.

I'm very impressed with the work you have done and I bet there will be several area yuo cover we will not e.g. the summary view you have.

Sean

**Comment by Jason3S** [comment]
Got it.  Thank you for keeping me updated.

**Comment by seanmcbreen** [comment]
Hi Jason,

A spellchecker is still some way for for VS Code (we are right now focused on [driving down some bug and other debt](https://github.com/Microsoft/vscode/issues/24111)) so I pulled the trigger on updating my extension to point to your one.

Let me know if you have any issues/concerns.  I updated the [README](https://marketplace.visualstudio.com/items?itemName=seanmcbreen.Spell) with a few details.

Sean
